dll/win32/msvcrt20/msvcrt20.c

```diff
@@ -82,6 +84,10 @@ DllMain(PVOID hinstDll, ULONG dwReason, PVOID reserved)
         if (!msvcrt_init_tls())
           return FALSE;
 
+        /* create heap */
+        if (!msvcrt_init_heap())
+            return FALSE;
```

Comment:
I think the function name is obvious enough to go without the top comment (ditto for the msvcrt40), otherwise lgtm.

Reply:
Removed the comments.